test/acceptance/test/pages/add_application_page.go

```diff
@@ -0,0 +1,34 @@
+package pages
+
+import (
+	"github.com/sclevine/agouti"
+)
+
+type AddAppPageElements struct {
+	AddAppHeader         *agouti.Selection
+	AppName              *agouti.Selection
+	AppNamespace         *agouti.Selection
+	AppRepoURL           *agouti.Selection
+	ConfigRepoURL        *agouti.Selection
+	PathToManifests      *agouti.Selection
+	Branch               *agouti.Selection
+	AutoMergeCheck       *agouti.Selection
+	SubmitButton         *agouti.Selection
+	AuthenticationButton *agouti.Selection
+}
+
+func GetAddAppPageElements(webDriver *agouti.Page) *AddAppPageElements {
+	addApplicationPage := AddAppPageElements{
+		AddAppHeader:         webDriver.FindByXPath(`//*[@id="app"]//nav//h2`),
+		AppName:              webDriver.FindByID("name"),
+		AppNamespace:         webDriver.FindByID("namespace"),
+		AppRepoURL:           webDriver.FindByID("url"),
```

Comment:
This will be very brittle.

Reply:
Do you plan to change the ID placement a lot? Or will the overall UI outlook change too much?

Reply:
Changed the xpaths to be as unique as possible